### ISSUE TITLE
Missing option for repositoryCredentials

### ISSUE THREAD

**Issue by waeltken** [cloudposse/terraform-aws-ecs-container-definition#19]
```json
"containerDefinitions": [
    {
        "image": "private-repo/private-image",
        "repositoryCredentials": {
            "credentialsParameter": "arn:aws:secretsmanager:region:aws_account_id:secret:secret_name"
        }
    }
]
```
https://docs.aws.amazon.com/AmazonECS/latest/developerguide/private-auth.html

**Comment by jc-asdf** [comment]
I've addressed this in #24; would love a review.  I'm using my fork of it in a project and it works well.

**Comment by aknysh** [comment]
addressed in #24
